### PR TITLE
Add yksom to CI

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

**Comment by jacob-hughes** [comment]
This can be merged once https://github.com/softdevteam/yksom/pull/228 lands.

File: .buildbot.sh [hunk 1]
```diff
@@ -16,3 +24,9 @@ ulimit -d $((1024 * 1024 * 8)) # 8 GiB
 
 /usr/bin/time -v python3 x.py test --stage 2 --config .buildbot.config.toml --exclude rustdoc-json --exclude debuginfo
 
+# Build and test yksom
+git clone --recursive https://github.com/softdevteam/yksom
+cd yksom
+rustup toolchain link alloy build/x86_64-unknown-linux-gnu/stage1
```
Comment (ltratt) [inline .buildbot.sh:30]: Should this line go before `cd yksom`?

Comment (jacob-hughes) [reply .buildbot.sh]: Oops. Yes. Fixed here: 4d03af851ba41a3e91ece9a389b95cd9e0ba26c1

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed, but this has to wait until I have fixed this https://ci.soft-dev.org/#/builders/1/builds/2931

**Comment by ltratt** [comment]
OK please ping me when it's ready.

**Comment by jacob-hughes** [comment]
When https://github.com/softdevteam/yksom/pull/228 lands. This one should be mergeable.

**Comment by jacob-hughes** [comment]
Closed in favour of https://github.com/softdevteam/alloy/pull/107

**Pull request closed without merging**